sdk/core/azure-core/test/ut/service_tracing_test.cpp

```diff
@@ -88,6 +88,22 @@ TEST(TracingContextFactory, SimpleServiceSpanTests)
     EXPECT_FALSE(contextAndSpan.Context.IsCancelled());
   }
 }
+
+TEST(TracingContextFactory, AttestationFactoryCtor)
```

Comment:
Do we want to have `Attestation` in the name of the test that live in core? Seems unnecessary.

```suggestion
TEST(TracingContextFactory, DeprecatedFactoryCtor)
```

Reply:
This ctor exists only for use by Attestation 1.0, and it's flagged as being deprecated, so maybe? I don't feel strongly.

Reply:
I'm pushing an update which improves comments and adds context for this test.